dcc-import-gene/src/test/java/org/icgc/dcc/imports/gene/writer/GeneWriterTest.java

```diff
@@ -1,127 +0,0 @@
-/*
- * Copyright (c) 2016 The Ontario Institute for Cancer Research. All rights reserved.
- *                                                                                                               
- * This program and the accompanying materials are made available under the terms of the GNU Public License v3.0.
- * You should have received a copy of the GNU General Public License along with                                  
- * this program. If not, see <http://www.gnu.org/licenses/>.                                                     
- *                                                                                                               
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY                           
- * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES                          
- * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT                           
- * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,                                
- * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED                          
- * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;                               
- * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER                              
- * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN                         
- * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-package org.icgc.dcc.imports.gene.writer;
-
-import static org.icgc.dcc.imports.core.util.Importers.getLocalMongoClientUri;
-
-import java.io.File;
-import java.io.IOException;
-
-import lombok.Cleanup;
-import lombok.SneakyThrows;
-import lombok.val;
-
-import org.icgc.dcc.common.test.mongodb.EmbeddedMongo;
-import org.icgc.dcc.imports.gene.writer.GeneWriter;
-import org.jongo.Jongo;
-import org.jongo.MongoCollection;
-import org.junit.Rule;
-import org.junit.Test;
-
-import com.fasterxml.jackson.databind.JsonNode;
-import com.github.fge.jsonschema.main.JsonSchema;
-import com.github.fge.jsonschema.main.JsonSchemaFactory;
-import com.github.fge.jsonschema.report.ValidationReport;
-import com.github.fge.jsonschema.util.JsonLoader;
-import com.mongodb.DB;
-import com.mongodb.MongoClient;
-import com.mongodb.MongoClientURI;
-
-public class GeneWriterTest {
```

Comment:
Curious why this was deleted. It can validate the produced JSON docs against a JSON Schema.